one-ui/ui/src/components/entities/entity-tiles.tsx

```diff
@@ -55,7 +55,7 @@ const EntityTiles = (props) => {
                 return mapArtifacts;
                }
 
-              console.log('GET Mapping Artifacts API Called successfully!',response);
+              console.log('GET Mapping Artifacts API Called successfully!');
             }
```

Comment:
Can we get rid of these console.logs completely? Logging errors is still ok, and maybe we should try console.lerror for those.

Reply:
@bsrikan - Yes, we can definitely remove these console logs completely and keep only the error logs. Updated the PR with the same. Please review now.